app/assets/javascripts/regional_request.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
I had to recover my repo after a sudden corruption. This seems to have come from your commit 
[https://github.com/fmip/cats-web/commit/1c24bfcac735b967fb615362f08f021c5dafee9a](url) . We can remove the comments. Same reason for your other comment.